plugins/CoreHome/angularjs/content-block/content-block.directive.js

```diff
@@ -83,8 +83,13 @@
                         }
                     }
 
+                    var tabs = angular.element('.tabs');
```

Comment:
good question. I double check it. I wanted to put the `$('.tabs').tabs();` to the page in a script tag, but even if I put inside the `document ready` the html was empty, so could find the tabs... it was annoying...
but I check it again.

Reply:
You could use an angular.run block, I guess, but not all the page content will be filled out by then and it wouldn't help for ajax... It might require a new attribute directive that's used on elements like `<div piwik-tabs>`. Otherwise, could just do it for this directive by injecting `$element` and using that (ie, `$element.find('.tabs')`).

Reply:
can you please check if this is what you thought?

Reply:
that works :+1: